data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3447 @@
+			choice
+				`	"I think it is true. The Hai who gave this tribute said they hoped the Unfettered would turn from violence and be reconciled with them."`
+					goto true
+				`	"I'm sorry, but I don't think they will ever let you live among them, if it is your nature to be prone to violence."`
+			`	You have little experience reading Hai emotion, but the youth seems crestfallen. "You are certain?" he asks.`
+			choice
+				`	"I am certain. I can do nothing for you."`
+					goto refuse
+				`	"Perhaps you can come with me and find out, but don't get your hopes up."`
+					goto end
+			label true
+			`	"Will you take me to Hai-home?" he asks. "You are able to be outside Unfettered places. Can I go with you?"`
+			choice
+				`	"Yes, I will take you there."`
+					goto end
+				`	"Sorry, I can't risk angering the Unfettered by carrying a defector."`
+			label refuse
+			`	Disappointed, the youth leaves. You hope you made the right choice.`
+				decline
+			label end
+			`	You show the youth to one of your bunk rooms, and tell him to stay hidden there until you reach Hai-home.`
+				accept
+	
+	on visit
+		dialog `You look for the young Hai, but realize that he took a ride on one of your escorts! Better depart and wait for your escorts to arrive in this star system.`
+	on complete
+		payment 100000
+		dialog
+			`You need not have worried about the reception the Unfettered youth would receive here on Hai-home. Scarcely minutes after you hesitantly contact the Hai government, a happy crowd has gathered around your ship. When he steps out of the hatchway, he looks stunned at being welcomed so warmly. One of the Hai governors thanks you for transporting him, and pays you <payment>.`
+		"reputation: Hai" += 10
+
+
+
+phrase "Essential Unfettered Mission Fail"
+	word
+		`You have failed an essential Unfettered mission. You may want to reload.`
+
+
+
+mission "Unfettered: Neutral With The Hai"
+	name `Get forgiven by the Hai`
+	description `The Unfettered have asked you to go negotiate with the Hai to stop your conflict with them. Go to <destination>.`
+	source
+		attributes "unfettered"
+	destination "Mirrorlake"
+	clearance `You speak with the local authorities and they agree to meet with you.`
+	to offer
+		has "reputation: Hai" < 0
+		has "Unfettered Jump Drive 3: offered"
+	on offer
+		conversation
+			`On your promenade you find an Unfettered elder who chastises you as a father would to a misbehaving child, "What have you done to anger our brethren?" You begin to answer, but he immediately interrupts you, "Never mind, it matters not. What matters is that you settle this petty dispute you have with the Hai government right now! Or, in the future, cooperation with us will become difficult."`
```

Comment:
"On your promenade" is a turn of phrase that will set people off for being an obscure word usage.

Reply:
`Whilst wandering near the spaceport` feels better?